sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2573,20 +2568,22 @@
       }
     },
     "TwinProperties": {
-      "description": "Represents Twin properties",
+      "description": "The twin's desired and reported properties. The maximum depth of twin property objects is 10.",
```

Comment:
Is "twin property objects" right word? Also is it talking about additional properties?

Reply:
Yes, additionalProperties is the container which will have the actual values for desired/ reported properties: https://swagger.io/docs/specification/data-models/dictionaries/

Reply:
so basically the comment is conveying nested properties upto 10?

Reply:
Exactly, this is the document for reference: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-device-twins#tags-and-properties-format